src/main_thread/text_displayer/html/text_track_cues_store.ts

```diff
@@ -233,7 +233,7 @@ export default class TextTrackCuesStore {
 
     for (let cueIdx = 0; cueIdx < cuesBuffer.length; cueIdx++) {
       let cuesInfos = cuesBuffer[cueIdx];
-      if (start < cuesInfos.end) {
+      if (start + DELTA_CUES_GROUP < cuesInfos.end) {
```

Comment:
Isn't there a risk of false positive for very small text segments?

Reply:
I don't think so, the value is `1e-3`. If the end of existing segment is almost equal to the start of the cue to insert, we can consider than we want to push after the existing segment rather than replacing it